app/views/layouts/_page.html.erb

```diff
@@ -0,0 +1,92 @@
+<% content_for :head do %>
+  <meta charset="utf-8">
+  <meta name="theme-color" content="blue">
+  <link rel="manifest" href="manifest.json">
+  <link rel="stylesheet" href="/stylesheets/govuk-frontend.min.css">
+<% end %>
+
+<% content_for :header do %>
+  <div class="govuk-header__container app-width-container">
```

Comment:
I don't think you need this div as you are using `govuk_header`

Reply:
You're right! Removed!